stablehlo/dialect/StablehloOps.td

```diff
@@ -731,7 +732,7 @@ def StableHLO_ComplexOp: StableHLO_BinaryElementwiseOp<"complex", [Pure,
 }
 
 def StableHLO_DivOp : StableHLO_BinaryElementwiseOp<"divide",
-      [Pure, HLO_CompatibleOperandsAndResultType]> {
+      [Pure, HLO_CompatibleOperandsAndResultType, Elementwise], HLO_FpOrComplexTensor> {
```

Comment:
DivOp should allow integers as well: https://github.com/openxla/stablehlo/blob/main/docs/spec.md#divide.